### PR TITLE
Cleaned up the advanced launcher

### PR DESCRIPTION
**Pull request by BenStarmerSmith**
## Proposed changes

Cleaned up the advanced launcher so when running Bimanual it splits the launch files into folders.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [x] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Comment by BenStarmerSmith** [comment]
Teleop Sim:
Screenshot from 2022-09-08 12-02-03

Hand+Glove:
Screenshot from 2022-09-08 12-12-13

File: ansible/inventory/teleop/group_vars/simulation.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ reinstall: false # Flag to know if the docker container should be fully reinstal
 nvidia_docker: true # Define if nvidia-container-toolkit is used (defaul is false)
 glove: "shadow_glove" # Determines which glove is installed (haptx or shadow_glove)
 container_name: "teleop_{{ glove }}" # Name of the docker container
+bimanual: true # when doing sim we should always have it bimanual.
```
Comment (bdenoun) [inline ansible/inventory/teleop/group_vars/simulation.yml:8]: out of curiosity, why?

**Review by bdenoun — APPROVED** [review]

File: ansible/inventory/teleop/group_vars/simulation.yml (hunk 1 above)
Comment (BenStarmerSmith) [reply ansible/inventory/teleop/group_vars/simulation.yml:8]: We always create bimanual icons for teleop in simulation, this just makes sure these new folders get created :)

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml [hunk 2]
```diff
@@ -100,14 +100,14 @@
     name: products/common/hand-manual
   when: customer_key is defined and customer_key | length > 0
 
-- name: Create shared icons
-  include_tasks: shared-roles.yml
-
 - name: Set default biotacs to false if Shadow Glove is being used (user can still override)
   set_fact:
     biotacs: false
   when: glove=="shadow_glove"
```
Comment (maxzieba) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml:106]: This can be removed right? We don't use biotacs anymore

**Review by maxzieba — APPROVED** [review]

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml (hunk 2 above)
Comment (BenStarmerSmith) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml:106]: Do we not at all?

**Pull request merged**